Makefile

```diff
@@ -217,7 +217,7 @@ release-docker: ### builds and produces the release packaging/tarball for TCE in
 		-w /go/src/community-edition \
 		-v ${PWD}:/go/src/community-edition \
 		-v /tmp:/tmp \
-		golang:1.16.2 \
+		golang:1.17.6 \
```

Comment:
Should we just use `golang:1.17` instead of fixing to a particular version? I mean, it's not bad to get the latest golang patch version from `1.17.x`, but yeah, releasing TCE with `release-docker` multiple times could produce different binaries with different behavior if done that way which is a con

Reply:
I think pinning to a patch and having reproducible builds is probably more desirable here vs using a docker base image with a floating patch version